### ISSUE TITLE
Accessibility service crash (NameNotFound)

### ISSUE THREAD

**Issue by ignoramous** [celzero/rethink-app#428]
```
<<< log_count = 1 >>>
[12-21 20:56:27.297 6408:6408 E/AndroidRuntime]
FATAL EXCEPTION: main
Process: com.celzero.bravedns, PID: 6408
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:549)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:994)
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	... 1 more
Caused by: android.content.pm.PackageManager$NameNotFoundException: tv.twitch.android.app
	at android.app.ApplicationPackageManager.getApplicationInfoAsUser(ApplicationPackageManager.java:427)
	at android.app.ApplicationPackageManager.getApplicationInfo(ApplicationPackageManager.java:408)
	at com.celzero.bravedns.util.BackgroundAccessibilityService.getEventUid(BackgroundAccessibilityService.kt:165)
	at com.celzero.bravedns.util.BackgroundAccessibilityService.handleAccessibilityEvent(BackgroundAccessibilityService.kt:143)
	at com.celzero.bravedns.util.BackgroundAccessibilityService.onAccessibilityEvent(BackgroundAccessibilityService.kt:104)
	at android.accessibilityservice.AccessibilityService$2.onAccessibilityEvent(AccessibilityService.java:1635)
	at android.accessibilityservice.AccessibilityService$IAccessibilityServiceClientWrapper.executeMessage(AccessibilityService.java:1824)
	at com.android.internal.os.HandlerCaller$MyHandler.handleMessage(HandlerCaller.java:44)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:225)
	at android.app.ActivityThread.main(ActivityThread.java:7563)
	... 3 more
```

At:

https://github.com/celzero/rethink-app/blob/161674b1cdbdd72fefee5edd953179a637e00f6f/app/src/main/java/com/celzero/bravedns/util/BackgroundAccessibilityService.kt#L165

**Comment by hussainmohd-a** [comment]
In this case, the accessibility services received event from the package which is not part of _Android's packageInfo_. Made changes to use the Utilities `getApplicationInfo` where the error handling in taken care. Now null will be returned from `getEventInfo` instead of exception.

**Comment by ignoramous** [comment]
Fixed: #380 

Release: `v053i`
